controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ControllerMaintenance.java

```diff
@@ -86,6 +88,7 @@ public ControllerMaintenance(MaintainerConfig maintainerConfig, ApiAuthorityConf
         costReportMaintainer = new CostReportMaintainer(controller, Duration.ofHours(2), reportConsumer, jobControl, nodeRepositoryClient, Clock.systemUTC(), selfHostedCostConfig);
         resourceMeterMaintainer = new ResourceMeterMaintainer(controller, Duration.ofMinutes(60), jobControl, nodeRepositoryClient, Clock.systemUTC(), metric, resourceSnapshotConsumer);
         nameServiceDispatcher = new NameServiceDispatcher(controller, Duration.ofSeconds(10), jobControl, nameService);
+        billingMaintainer = new BillingMaintainer(controller, Duration.ofDays(15), jobControl, billingHandler);
```

Comment:
With our current release frequency this will never run. The initial execution delay of each maintainer is equal to the interval.